src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (...num) => {
+    if (num.length === 0) {
+      const tmp = sum;
+
+      sum = 0;
+
+      return tmp;
+    }
+
+    sum = num.reduce((el1, el2) => el1 + el2, sum);
```

Comment:
The use of `reduce` here is unnecessary since you will only ever have one number at a time. You can simplify this by adding the single argument directly to `sum`.